members/README.md

```diff
@@ -89,28 +92,54 @@ Returns all inactive/idle members in the system.
     - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'Something went wrong. Please contact admin' }`
 
 ## POST /members/intro/:username
-Creates new request for member introduction 
+
+Creates new request for member introduction
 
 - **Params**\
-    _Required_: `username=[string]`
+   _Required_: `username=[string]`
 - **Query**\
-    None
+   None
 - **Body**\
-    `{<recruiter_object>}`
+   `{<recruiter_object>}`
 - **Headers**\
-    None
+   None
 - **Cookie**\
-    None
+   None
 - **Success Response:**
-    - **Code:** 200
-        - **Content:** 
-        ```
-        {
-            message: 'Request Submission Successful!!'
-            id: <new unique submission id>
-        }
-        ```
+
+  - **Code:** 200
+    - **Content:**
+    ```
+    {
+        message: 'Request Submission Successful!!'
+        id: <new unique submission id>
+    }
+    ```
+
+- **Error Response:**
+  - **Code:** 500
+    - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
+
+## Patch /members/archiveMembers/:username
+
+Archive members / users
+
+- **Params**\
+   _Required_: `username=[string]`
+- **Query**\
+   None
+- **Body**\
+   `{reason: <string>}`
+- **Headers**\
+   None
+- **Cookie**\
+   None
+
+  - **Success Response:**
+  - **Code:** 204
```

Comment:
Better than 204 we can send 200 with the update value in the response !

Reply:
Yes, updated the response to status `200` having response

```js
{"message":"User archived successfully","reason":"lorem ipsum"}
```